src/cpptests/t_llapi.cpp

```diff
@@ -286,7 +286,7 @@ TEST_F(LLApiTest, testRanges) {
     RediSearch_SpecAddDocument(index, d);
   }
 
-  RSQNode* qn = RediSearch_CreateLexRangeNode(index, FIELD_NAME_1, "MarkN", "MarkX");
+  RSQNode* qn = RediSearch_CreateLexRangeNode(index, FIELD_NAME_1, "MarkN", "MarkX", 1, 1);
```

Comment:
I'd prefer flags, e.g. `REDISEARCH_RANGE_EXCLUDE_MIN` and `REDISEARCH_RANGE_INCLUDE_MAX`. Default should still be as before, where the begin is inclusive but the end is exclusive.

Reply:
This is how the LEXNODE is defines I want to keep them the same..